### DISCUSSION TITLE
Kaspersky warns for a trojan on pixi.exe

### DISCUSSION THREAD

**Discussion by biofsphere** [prefix-dev/pixi#1630] [General]
As I was eager to test Pixi this morning, after installing it according to documentation, Kaspersky reported a nasty trojan at pixie.exe. Is this a false positive? I have the MD5 if useful to anyone.

**Comment by pavelzw** [comment]
this is very likely a false positive. as long as you downloaded it from the official sources you should be fine

**Reply by biofsphere** [reply]
I ran the Powershell command suggested in the documentation. I assume it was downloaded from the official source. For someone who lost a whole HD of data to some hacker, "very likely" sounds not worth the risk. Is there a way to get to the "100% sure" mark? Is it possible to have a SHA256 check? Can Kaspersky be informed by someone in the team, if this is a false positive? I am not a security expert, just a freaking out security paranoic hobbyist dev =)

**Reply by ruben-arts** [reply]
@biofsphere We can look into sharing a SHA256 with the binaries. If you've used the Powershell command I can't see how that would not get the binaries build by use and put in GitHub. 

Send Kaspersky an email.

**Reply by ruben-arts** [reply]
That said, it would be good if you could share the checksums for your downloaded binary.

**Reply by biofsphere** [reply]
Thank you for your suggestions. Panic led me to sanitize my system and uninstall pixi entirely. But I will go over it again and follow your lead. Poetry is great, but this is the only package manager that I got GDAL installed without much hassle.

**Reply by ruben-arts** [reply]
I can understand that you did that. Hope we can find a solution.

You can also use `winget`: `winget install prefix-dev.pixi`. Which is an official Microsoft cli tool installer. They redistribute our binaries but do some verification on the binary as well.

The downside of installing with another package manager is that you have to use that manager for updates in the future.

**Comment by biofsphere** [comment]
The easiest way I found is to exclude the false-positive file from scanning, although, that isn't a security best practice approach. But that's what Kaspersky support will tell you to do.

**Comment by ruben-arts** [comment]
As expected, I'm not getting a response from Kaspersky. This is unfortunately out of our control. Hopefully, you flagging it as false positive will also help for others!
